### ISSUE TITLE
How to use Remotion with TailwindCSS?

### ISSUE THREAD

**Issue by frencojobs** [remotion-dev/remotion#449]
Tailwind was mentioned in the Remotion intro video but I couldn't find an example or any mention in the documentation on how to use Remotion with Tailwind properly.

### What I've Tried So Far

Obviously, attempting to use Tailwind just by installing Tailwind and importing the CSS file won't work. So I tried to include PostCSS in the Webpack config and get it to work but *only with an angle*.

After setting up Tailwind with PostCSS as usual (installing Tailwind, PostCSS, Autoprefixer, and importing `tailwind.css` file), when I put this config into `webpack config > module > rules` as guided in SCSS setup, I got this error.
```js
// part of config
{
	test: /\.css$/i,
	use: [
		{loader: 'style-loader'},
		{loader: 'css-loader'},
		{loader: 'postcss-loader'}, // yes postcss-loader was installed first
	],
},
```
```sh
# error
ERROR in ./node_modules/@remotion/cli/styles/styles.css (./node_modules/css-loader/dist/cjs.js!./node_modules/style-loader/dist/cjs.js!./node_modules/css-lo
ader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./node_modules/@remotion/cli/styles/styles.css)                                                  
Module build failed (from ./node_modules/css-loader/dist/cjs.js):                                                                                           
CssSyntaxError
# ...
```

The *angle* was when I removed the first two loaders, it started working well. 
```js
// part of config
{
	test: /\.css$/i,
	use: [
		{loader: 'postcss-loader'}
	],
},
```
With this config, I've tested that both normal CSS contents and TailwindCSS stuff work as expected. Hot reload also works and Tailwind config is also loaded as expected. Everything works just fine.

My question is that is there any problem with the fact that I removed those two loaders? (I mean it works fine without them but there have to be some shortcomings, right?)

Plus, I wish there is an example repository for guided setups with multiple frameworks like [twin.macro](https://github.com/ben-rogerson/twin.examples/) does.

Thanks! I really like Remotion.

**Comment by JonnyBurger** [comment]
Hey Frenco! Thanks a lot for raising this issue and explaining everything thoroughly - indeed we have not documented it and it's not necessarily super straightforward.

We have now added instructions and example configuration on how to add TailwindCSS to Remotion.
In the longterm we want to have a template for it as well, but for it we probably want to rewrite the whole Hello World to use TailwindCSS. Until then, I hope the new instructions serve you well!

https://www.remotion.dev/docs/webpack/#enable-tailwindcss-support

**Comment by LucasIcarus** [comment]
@JonnyBurger  

![image](https://user-images.githubusercontent.com/15810641/131327130-d992f967-c36d-476e-b5a9-00c4e28efeb7.png)

Import stylesheet import to `src/index.tsx` will cause hmr warning ("Ignored an update to unaccepted module") because `src/index.tsx` is main file to webpack. 

Move stylesheet import to `src/Video.tsx` works fine.

**Comment by JonnyBurger** [comment]
Thanks @LucasIcarus for letting us know of the mistake! I fixed this in db907ab03.
